.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -11,7 +11,7 @@ assignees: ''
 
 ## Expected Behavior
 
-<GIVEN>
+GIVEN
```

Comment:
Why has this been changed? Please revert, if it was changed unintentionally.

Reply:
Because Sonarlint threw an error that the tag was not closed and therefore I couldn't commit it.